Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart for Docker Registry
 name: docker-registry
-version: 1.10.1
+version: 1.11.0
```

Comment:
Generally, it's a best practice to avoid tethering a version increment to a PR's changeset.

Versioning is a responsibility of the release team, and there may be other PRs going on in parallel that may otherwise have conflicting version increments. It's helpful for any project, though, to *suggest* whether a changeset warrants a patch, minor, or major version increment in the PR's description. Manual version increments could conflict, too, in projects with automated releases, like [semantic release](https://github.com/semantic-release/semantic-release).